swagger.yaml

```diff
@@ -48,6 +51,11 @@ paths:
           type: string
           required: true
           description: component version
+        - in: query
```

Comment:
security-vendor is a header field not query param.

Reply:
Ack!